release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -797,6 +797,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 You can attempt to resolve the issue with the steps in this link:https://kb.vmware.com/s/article/2002779[VMware KBase article]. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/5785341[[UPI vSphere\] Node scale-up doesn't work as expected]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918383[*BZ#1918383*])
 
+* An Open Virtual Network (OVN) bug causes persistent connectivity issues to Octavia load balancers. This problem affects Neutron subnets, or OpenShift namespaces, at random. Because of this bug, {product-title} 4.8 deployments that use Kuryr SDN are not recommended on {rh-openstack-first} 16.1 with OVN, and OVN Octavia, configured. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937392[*BZ#1937392*])
```

Comment:
"with OVN, and OVN Octavia, configured."

^ Are the commas needed here?

Reply:
"causes persistent connectivity issues to Octavia load balancers."

"to" or "with"? :thinking: